source/features/search/store.ts

```diff
@@ -136,7 +136,11 @@ export class SearchStore extends Store {
       const hasFoundBlock = blocksResult && blocksResult.data.blocks.length > 0;
       const hasFoundEpoch = epochsResult && epochsResult.data.epochs.length > 0;
       if (hasFoundBlock && !hasFoundEpoch) {
-        path = `/block?number=${params.number}`;
+        // https://github.com/kulshekhar/ts-jest/issues/1283
+        path = `/block?id=${blocksResult.data &&
+          blocksResult.data.blocks &&
+          blocksResult.data.blocks[0] &&
+          blocksResult.data.blocks[0].id}`;
```

Comment:
That would have been a perfect use case for optional chaining ;)

Reply:
I did, then tripped over the ts-jest 'blocker', refactored to just get it through. We can update it without any real issue